.github/workflows/nf-test.yml

```diff
@@ -101,16 +91,18 @@ jobs:
   nf-test:
     runs-on:
       - runs-on=${{ github.run_id }}
-      - runner=4cpu-linux-x64
-      - image=ubuntu22-full-x64
```

Comment:
don't we need to specify an image?

Reply:
According to the docs: 
>By default, the image defaults to ubuntu22-full-x64. If you want to use arm64 family instance types, and don’t specify a default runner name as base (e.g. runner=2cpu-linux-arm64) you will also need to override the image label.

Reply:
🤔 maybe because we do specify a runner at [line 94 ](https://github.com/nf-core/modules/pull/7747/files#diff-a6e08ed61fd370dba7712991afc0f04ab727f2c501c3ca2e0da629142c4e7bbaR94)👍

Reply:
yep, image is not needed. I just always add this line, because I will probably switch to ubuntu24 soon to mirror github